optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/OptaPlannerBuildTimeConfig.java

```diff
@@ -40,6 +43,14 @@
     @ConfigItem
     Optional<String> solverConfigXml;
 
+    /**
+     * A classpath resource to read the solver score DRL.
+     * Defaults to "{@link #DEFAULT_CONSTRAINTS_DRL_URL}".
+     * This property must NOT be defined in case a {@link ConstraintProvider} implementation exists.
+     */
+    @ConfigItem
+    Optional<String> constraintsDrl;
```

Comment:
(important)

Should this be constraintsDrl or scoreDRL? (regardless of what we decided before)

I'd argue there is no point in naming things in application.properties and solverConfig.xml differently if they do the same thing. It just creates unneeded complexity and takes a user's brainpower unnecessarily. Wdyt?

Reply:
On the other hand, it does indeed default to `constraints.drl`, something we changed intentionally from `scoreRules.drl`

Reply:
Worth discussing I think on bluejeans. Let me finish the review first.

Reply:
Agreed to change the property name to `optaplanner.constraints-drl` later(TODO: file Jira).
`optaplanner.score-drl`
The default stays `constraints.drl`.

Reply:
https://issues.redhat.com/browse/PLANNER-2285